src/components/attendees/styles.css

```
@@ -0,0 +1,94 @@
+table {
+	border-collapse: collapse;
+	border-spacing: 5px;
+}
+
+tr {
+	text-align: start;
+}
+th {
+	border-bottom: 2px solid #e8e8e8;
+	min-width: 50px;
+	white-space: nowrap;
+	font-weight: 700;
+	font-size: 20px;
+	color: #4b4b4b;
+	padding: 5px;
+	padding-right: 10px;
+	padding-left: 10px;
+}
+
+td {
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+
+	border-bottom: 2px solid #e8e8e8;
+	font-weight: 600;
+	font-size: 20px;
+	line-height: 24px;
+	color: #7f7f7f;
+	padding: 10px;
+
+	text-align: center;
+}
+
+#openUser {
+	width: 171px;
+	height: 39px;
+
+	background: #f5f5f5;
+	border: 2px solid #e8e8e8;
+	border-radius: 10px;
+}
+#openUser:hover {
```

```suggestion

#openUser:hover {
```
I think we should have a new line here for consistency.